### PR TITLE
Fix jk_selection

### PR DESCRIPTION
**Pull request by ShengYenPeng**
the setup of bSymbolArrBrkpt is not what we expected.
I modifiy the setup and then it works well.

### CODE REVIEW

**Comment by ShengYenPeng** [comment]
ref #149

File: test/test-bopomofo.c [hunk 1]
```diff
@@ -1370,8 +1370,8 @@ void test_interval()
 
 void test_jk_selection()
 {
-#if 0
-FIXME: libchewing is broken in this case
+#if 1
+//FIXME: libchewing is broken in this case
     ChewingContext *ctx;
```
Comment (czchen) [inline test/test-bopomofo.c]: Please remove `#if` and comment here.

**Comment by czchen** [comment]
Hi @ShengYenPeng 

Please help to rebase this PR since it contains commits (PR #222) that does not belong to this PR, and there are so many duplicated commits in this PR.

**Comment by ShengYenPeng** [comment]
Hi @czchen  

I have rebase and squash duplicate commits

**Comment by czchen** [comment]
Thanks for your contributions.

**Comment by ShengYenPeng** [comment]
Should the #149 be closed?

**Pull request merged**